infection.json

```diff
@@ -0,0 +1,17 @@
+{
+  "ignoreMsiWithNoMutations": true,
+  "logs": {
+    "text": ".build/infection/infection-log.txt"
+  },
+  "minCoveredMsi": 100,
+  "minMsi": 100,
+  "phpUnit": {
+    "configDir": "."
+  },
+  "source": {
+    "directories": [
+      "src"
+    ]
```

Comment:
When you declare test code in a separate directory, managing exclusions for `infection/infection` in `infection.json` is unnecessary.